baclahauC/jsons/jsonCIDs.txt

```diff
@@ -1,25 +1,18 @@
 test1.json : QmSCMeGh17KJWA9VHPftWqA1xdWM7McVtt3azxH3aenp1X
 test2.json : QmZhCRWWv8x6zJKhUKwnDNzerWoUaAjyeD4aSY8tGz6qE4
 test3.json : QmT3sHvzDBV53WqY2Z2Gt5U9zFPZYHKHMSKghbbkxt3zsM
+claim.jsom : QmTFcE6qwbpN5YozkR7u8yztZGYYjAztuxeF3VQMpAJwh1
```

Comment:
I think here is a typo. It should be `json`, right? Instead of `jsom`

Reply:
yea it should, good catch